### PR TITLE
Fix errcheck issues in server/channels/app/imaging/utils_test.go

### PR DESCRIPTION
**Pull request by samarth29jc**
#### Summary

Fix errcheck issues in utils_test.go

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28947
JIRA: [MM-61256](https://mattermost.atlassian.net/browse/MM-61256)

#### Release Note

```release-note

```

[MM-61256]: https://mattermost.atlassian.net/browse/MM-61256?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: server/channels/app/imaging/utils_test.go [hunk 1]
```diff
@@ -111,4 +118,4 @@ func TestFillImageTransparency(t *testing.T) {
 
 		require.Equal(t, expectedImg, inputImg)
 	})
-}
+}
```
Comment (hanzei) [inline server/channels/app/imaging/utils_test.go:121]: Please remove the unrelated change

**Review by hanzei — CHANGES_REQUESTED** [review]
One request, otherwise LGTM

File: server/channels/app/imaging/utils_test.go (hunk 1 above)
Comment (samarth29jc) [reply server/channels/app/imaging/utils_test.go]: Done

**Review by hanzei — APPROVED** [review]
:rocket:

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
@samarth29jc can you fix the lint issues? Should be pretty simple.

**Comment by samarth29jc** [comment]
> @samarth29jc can you fix the lint issues? Should be pretty simple.

Sure.
